### PR TITLE
Properties added

### PR DESCRIPTION
**Pull request by willrnch**
This add the properties endpoint responsible of retrieving the data to custom the UI. Data values are stored in the configurator.

### CODE REVIEW

File: config/policies.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports.policies = {
 
   '*': 'isAuthorized',
 
-  'PropertyController': {
-    find: true
+  PropertyController: {
+    '*': true
```
Comment (dynamiccast) [inline config/policies.js]: Only `find` should be accesible without authentication

**Comment by dynamiccast** [comment]
Depends on https://github.com/Nanocloud/nanocloud/pull/14

**Comment by dynamiccast** [comment]
@Costent This branch has conflict. Can you rebase please ?

File: api/controllers/PropertyController.js [hunk 2]
```diff
@@ -1,14 +1,51 @@
 /**
- * PropertyController
+ * PropertyController is responsible of the installation properties. Those are
+ * publicly accessible (wihtout authentication) and are firstly used to custom
```
Comment (dynamiccast) [inline api/controllers/PropertyController.js]: wihtout => without
to custom => to customize

File: config/env/development.js [hunk 3]
```diff
@@ -19,6 +19,12 @@ module.exports = {
 
   models: {
      connection: 'postgresDevelopment'
-  }
+  },
 
+  nanocloud: {
```
Comment (dynamiccast) [inline config/env/development.js:24]: This should be in the production environment file too.

Comment (willrnch) [inline config/env/development.js:24]: done

**Comment by dynamiccast** [comment]
Closed in favor of https://github.com/Nanocloud/nanocloud/pull/18

**Pull request closed without merging**